apps/chat/src/components/Chat/Playback/PlaybackIcon.tsx

```diff
@@ -12,14 +12,44 @@ interface Props {
   strokeWidth?: number;
 }
 
-export const PlaybackIcon = ({ isCustomTooltip, ...props }: Props) => {
+export const PlaybackIcon = ({
+  isCustomTooltip,
+  size = 24,
+  strokeWidth = 1,
+  ...props
+}: Props) => {
   const { t } = useTranslation(Translation.Chat);
 
+  const innerSize = (size * 7) / 8;
```

Comment:
I think it would be good to assign these numbers (7 and 8) to some constants to improve code readability.